package.json

```diff
@@ -47,7 +47,8 @@
     "less-plugin-clean-css": "^1.5.0",
     "npm-run-all": "^4.0.1",
     "standard": "^8.6.0",
-    "wintersmith": "^2.3.6"
+    "wintersmith": "^2.3.6",
+    "babel-plugin-transform-es2015-modules-umd": "^6.22.0"
```

Comment:
Or is it actually the opposite way round? Listen, this is definitely not the way we will be getting anywhere.